src/lib/types.ts

```diff
@@ -0,0 +1 @@
+type Units = 'BMP';
```

Comment:
units will not always be the same - I was only using a single parameter unit as `BPM` but they can be many others. So you should remove this file